components/TextFields/src/ContainedInputView/MDCBaseTextField.m

```diff
@@ -124,6 +164,16 @@ - (CGRect)adjustTextAreaFrame:(CGRect)textRect
   return CGRectMake(CGRectGetMinX(textRect), minY, CGRectGetWidth(textRect), systemDefinedHeight);
 }
 
+- (void)layOutLabel {
```

Comment:
Suggestion: call this postlayoutLabel?

Reply:
This is a placeholder, I will change it in the next PR